2025-03-03 13:41:40,180 - Ошибка распознавания: закрой заход -> tracking_color 1 слежу по камере
2025-03-03 13:49:57,108 - Ошибка распознавания: зажми заходят -> tracking_color 1 слежу по камере
2025-03-03 14:09:42,703 - Ошибка распознавания: закройте отпад -> tracking_color 1 слежу по камере
2025-03-03 17:31:30,637 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 17:31:30,907 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 17:31:37,646 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 17:31:38,053 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 17:31:53,863 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 17:31:54,273 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 17:37:45,305 - Ошибка распознавания: открой заход -> moveCam gcodeLock2Fing1 закрываю захват
2025-03-03 18:47:01,938 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 18:47:10,217 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-03 19:00:27,769 - Ошибка распознавания: словам -> tracking_color 1 слежу по камере
2025-03-03 19:01:07,671 - Ошибка распознавания: столам -> tracking_color 1 слежу по камере
2025-03-03 19:01:31,989 - Ошибка распознавания: салон -> tracking_color 1 слежу по камере
2025-03-03 19:01:59,517 - Ошибка распознавания: вам -> tracking_color 1 слежу по камере
2025-03-03 19:02:11,118 - Ошибка распознавания: салом -> tracking_color 1 слежу по камере
2025-03-03 19:02:24,045 - Ошибка распознавания: салам -> tracking_color 1 слежу по камере
2025-03-04 12:33:07,429 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-04 12:33:07,794 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-04 13:30:35,069 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-04 19:34:13,184 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-04 19:34:22,569 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:28:07,191 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:28:12,639 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:28:46,070 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:28:52,312 - Ошибка распознавания: вращая зет на -> move gcodeY перемещаю игрик
2025-03-11 13:29:04,773 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:29:10,872 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:30:07,074 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:30:10,655 - Ошибка распознавания: переместить икс на -> move gcodeZ перемещаю Z
2025-03-11 13:30:19,926 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:36:41,150 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:36:46,109 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:36:55,504 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:37:08,987 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:37:22,152 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:37:26,038 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:38:27,645 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:38:31,791 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:40:08,627 - Ошибка распознавания: открой заклад -> tracking_color 1 слежу по камере
2025-03-11 13:45:34,453 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:45:39,804 - Ошибка распознавания: перенести игрек на минут -> move gcodeY передвигаю плечо на 
2025-03-11 13:45:53,779 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:51:00,599 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:51:05,433 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:51:23,542 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:51:27,434 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:51:41,239 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:51:45,947 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:52:00,719 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:52:04,020 - Ошибка распознавания: перенести икс на -> move gcodeY передвигаю плечо на 
2025-03-11 13:52:24,033 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:52:34,552 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:52:46,090 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:52:56,301 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:55:31,000 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:55:46,889 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:55:51,244 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:56:00,831 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:56:12,552 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:56:19,027 - Ошибка распознавания: перевести игры к на -> move gcodeZ перемещаю Z
2025-03-11 13:56:39,387 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:56:59,734 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:57:16,751 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:57:20,870 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 13:57:34,353 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:00:23,528 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:00:33,872 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:00:53,801 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:01:06,237 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:01:25,394 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:01:31,481 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:01:52,857 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:01:57,812 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:02:26,241 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:02:43,843 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:02:47,743 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:03:01,249 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:03:05,675 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:03:17,388 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:03:45,147 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:03:49,495 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:04:28,691 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:04:33,110 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:04:51,552 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:05:01,497 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:36:32,188 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:36:38,715 - Ошибка распознавания: переместить кисть на -> move gcodeZ перемещаю Z
2025-03-11 14:36:55,168 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:37:18,741 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:37:24,987 - Ошибка распознавания: перенести есть на -> move gcodeY передвигаю плечо на 
2025-03-11 14:37:37,988 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:37:41,298 - Ошибка распознавания: переместить кисти на -> move gcodeZ перемещаю Z
2025-03-11 14:37:53,023 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:37:57,486 - Ошибка распознавания: перевести кисть на -> move gcodeZ перемещаю Z
2025-03-11 14:38:07,822 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:38:42,212 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 14:38:51,808 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:15:21,885 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:19:06,733 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:19:20,918 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:19:23,712 - Ошибка распознавания: перевести дабл ю на -> move gcodeZ перемещаю Z
2025-03-11 15:21:12,948 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:21:21,775 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:22:09,059 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:22:22,052 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:22:28,960 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:22:47,266 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:22:54,012 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:23:51,948 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:24:06,061 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:37:40,986 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:37:57,899 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:38:01,490 - Ошибка распознавания: вращая и кисть на -> move gcodeW перемещаю кисть
2025-03-11 15:38:20,778 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:38:28,572 - Ошибка распознавания: вращайте есть на -> move gcodeW перемещаю кисть
2025-03-11 15:38:40,226 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:38:52,713 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:38:56,453 - Ошибка распознавания: вращает кисть на -> move gcodeW перемещаю кисть
2025-03-11 15:39:07,038 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:39:11,819 - Ошибка распознавания: вращая кисть на -> move gcodeW перемещаю кисть
2025-03-11 15:39:21,943 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:39:34,516 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:39:49,509 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:39:52,806 - Ошибка распознавания: вращая есть на -> move gcodeW перемещаю кисть
2025-03-11 15:40:02,031 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:40:14,913 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:40:33,056 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:40:46,128 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:41:01,528 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:41:08,203 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:41:39,473 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:42:00,279 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:42:22,287 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:42:45,790 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:43:32,486 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:43:43,945 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:43:48,858 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-11 15:46:16,452 - Ошибка распознавания: выключить питание -> tracking_color 1 слежу по камере
2025-03-11 15:46:34,517 - Ошибка распознавания: выключи питания -> tracking_color 1 слежу по камере
2025-03-11 15:46:59,437 - Ошибка распознавания: включи питания -> move gcodeFANon включаю вентиляторы
2025-03-17 15:40:12,148 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:40:15,185 - Ошибка распознавания: вращаясь захват на -> move gcodeZ перемещаю Z
2025-03-17 15:40:51,246 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:41:04,678 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:41:07,788 - Ошибка распознавания: возвращай кисть на -> move gcodeW перемещаю кисть
2025-03-17 15:41:18,307 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:41:32,749 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:41:36,250 - Ошибка распознавания: согни кисть на -> move gcodeV вращаю кисть
2025-03-17 15:41:48,473 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:42:00,269 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:42:09,830 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-17 15:42:13,124 - Ошибка распознавания: возвращайтесь на -> move gcodeZ перемещаю Z
2025-03-17 22:57:35,942 - Ошибка распознавания: сосал -> tracking_color 1 слежу по камере
2025-03-17 22:58:02,630 - Ошибка распознавания: всосал -> tracking_color 1 слежу по камере
2025-03-17 22:58:45,123 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "comtypes\_comobject.py", line 176, in call_without_this
  File "comtypes\client\_events.py", line 125, in error_printer
  File "pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
TypeError: 'int' object is not subscriptable
2025-03-17 23:28:31,600 - Ошибка распознавания: здорово столом -> tracking_color 1 слежу по камере
2025-03-20 13:31:52,797 - Ошибка распознавания: включи ручное управление -> move gcodeFANon включаю вентиляторы
2025-03-20 15:20:55,296 - Ошибка распознавания: переключи ручной режим -> tracking_color 1 слежу по камере
2025-03-20 15:21:13,800 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-20 15:21:20,554 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-20 16:01:24,918 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-20 16:01:39,020 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-20 18:12:52,869 - Ошибка распознавания: шли по камеры -> move gcodeClock поворот по часовой
2025-03-20 19:43:32,792 - Ошибка распознавания: же блядь я ебал блять ну типа я уже устал второй день сидеть в там кабинете читать -> tracking_color 1 слежу по камере
2025-03-20 20:00:46,057 - Ошибка распознавания: арии виды арчи -> tracking_color 1 слежу по камере
2025-03-20 20:01:27,218 - Ошибка распознавания: ари виды арчи -> tracking_color 1 слежу по камере
2025-03-20 22:57:00,386 - Ошибка распознавания: не знаю что обойдя а вместе -> tracking_color 1 слежу по камере
2025-03-28 16:54:32,076 - Ошибка распознавания: сосал -> tracking_color 1 слежу по камере
2025-03-28 17:02:54,138 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:03:48,458 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:04:19,317 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:04:24,262 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:04:37,506 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:04:43,485 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:04:52,819 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-03-28 17:05:03,555 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-04 13:46:56,728 - Ошибка распознавания: танцуй -> tracking_color 1 слежу по камере
2025-04-10 14:48:34,298 - Ошибка распознавания: обвнт манпулятора выключ птане -> tracking_color 1 слежу по камере
2025-04-10 15:04:58,736 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:05:16,075 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:05:28,610 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:05:51,490 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:06:02,487 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:06:15,431 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:06:28,903 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:06:53,092 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:07:03,238 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:07:16,730 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:07:27,313 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:07:49,082 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:08:07,656 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:08:12,149 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:08:23,754 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:08:33,510 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:08:44,258 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:24:16,342 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:24:28,673 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:24:32,631 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:24:41,263 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:25:25,855 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:25:39,901 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:26:06,657 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:26:15,819 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:26:25,609 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:26:35,690 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:27:23,758 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:27:40,152 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:27:44,621 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:27:54,834 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:28:10,885 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:28:38,555 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:28:48,951 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:28:59,417 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:29:13,728 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:29:18,100 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:29:28,281 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:29:32,202 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-10 15:39:53,373 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-15 12:28:30,420 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-16 13:16:48,475 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
2025-04-16 13:16:59,046 - Exception in _ISpeechVoiceEvents.Word implementation:
Traceback (most recent call last):
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\_comobject.py", line 176, in call_without_this
    result = mth(*inargs)
             ^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\comtypes\client\_events.py", line 125, in error_printer
    return func(*args, **kw)
           ^^^^^^^^^^^^^^^^^
  File "D:\voice_assistent\venv\Lib\site-packages\pyttsx3\drivers\sapi5.py", line 182, in _ISpeechVoiceEvents_Word
    current_word = current_text[char:char + length]
                   ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not subscriptable
